pkg/agent/agent.go

```diff
@@ -557,5 +557,9 @@ func applyResConfigToCentralConfig(cfg *config.CentralConfiguration, resCfgAddit
 		log.GlobalLoggerConfig.Level(logLevel).Apply()
 	}
 
-	cfg.TeamName = resCfgTeamName
+	// If resource team name is not blank, apply it
+	if resCfgTeamName != "" {
```

Comment:
I still think you want the cfg.TeamName == "" in this condition.

What would happen if both are set, the resource one would take precedence when it should not.

Reply:
crap... sorry i got balls in the air again.